theme/basics/template-hierarchy.md

```diff
@@ -1,419 +1,184 @@
 
 1.  `home.php`
 2.  `index.php`
 
-<!--
 Note: If `front-page.php` exists, it will override the `home.php` template.
--->
-注意: `front-page.php` が存在する場合は、 `home.php` テンプレートに優先します。
 
-<!-- 
 ### Front Page display
--->
-### フロントページ表示
 
-<!--
 The `front-page.php` template file is used to render your site’s front page, whether the front page displays the blog posts index (mentioned above) or a static page. The front page template takes precedence over the blog posts index (`home.php`) template. If the `front-page.php` file does not exist, WordPress will either use the `home.php` or `page.php` files depending on the setup in Settings → Reading. If neither of those files exist, it will use the `index.php` file.
--->
-`front-page.php` テンプレートファイルはサイトのフロントページの表示に使用されます。フロントページは(上述の)ブログ投稿インデックスか、固定ページを表示します。フロントページのテンプレートは、ブログ投稿インデックス（`home.php`）のテンプレートよりも優先されます。`front-page.php` ファイルが存在しない場合、WordPress は 設定 > 表示設定 での設定に応じて `home.php` または `page.php` ファイルを使用します。もしこれらのファイルがどちらも存在しない場合は、`index.php` ファイルを使用します。
 
-<!--
 1.  `front-page.php` – Used for both “**your latest posts**” or “**a static page**” as set in the **front page displays** section of Settings → Reading.
 2.  `home.php` – If WordPress cannot find `front-page.php` and “**your latest posts**” is set in the **front page displays** section, it will look for `home.php`. Additionally, WordPress will look for this file when the **posts page** is set in the **front page displays** section.
 3.  `page.php` – When “**front page**” is set in the **front page displays** section.
 4.  `index.php` – When “**your latest posts**” is set in the **front page displays** section but `home.php` does not exist *or* when **front page** is set but `page.php` does not exist.
--->
-
-1. `front-page.php` - 設定 > 表示設定 セクションで「**フロントページの表示**」が「**最新の投稿**」または「**固定ページ**」どちらになっている場合でも使われます。
-2. `home.php` - WordPress が `front-page.php` を見つけられず、且つ「**フロントページの表示**」セクションで「**最新の投稿**」が設定されている場合、`home.php` を参照します。加えて、WordPress は「**フロントページの表示**」セクションで「**投稿ページ**」が設定されている場合にもこのファイルを参照します。
-3. `page.php` – 「**フロントページの表示**」セクションで**フロントページ**が設定されている場合。
-4. `index.php` – 「**フロントページの表示**」セクションで「**最新の投稿**」が設定されており、且つ `home.php` が存在しない場合。または**フロントページ**が設定されているが `page.php` が存在しない場合。
 
-<!--
 As you can see, there are a lot of rules to what path WordPress takes. Using the chart above is the best way to determine what WordPress will display.
--->
-このように、WordPressがどのような経路をとるかには多くのルールがあります。上の図を使用することは、WordPress が何を表示するかを特定する最良の方法です。
 
-<!--
 ### Privacy Policy Page display
--->
-### プライバシーポリシーページ表示
 
-<!--
 The `privacy-policy.php` template file is used to render your site’s Privacy Policy page. The Privacy Policy page template takes precedence over the static page (`page.php`) template. If the `privacy-policy.php` file does not exist, WordPress will either use the `page.php` or `singular.php` files depending on the available templates. If neither of those files exist, it will use the `index.php` file.
```

Comment:
<!--
The `privacy-policy.php` template file is used to render your site’s Privacy Policy page. The Privacy Policy page template takes precedence over the static page (`page.php`) template. If the `privacy-policy.php` file does not exist, WordPress will either use the `page.php` or `singular.php` files depending on the available templates. If neither of those files exist, it will use the `index.php` file.
-->
`privacy-policy.php` テンプレートファイルは、あなたのサイトのプライバシーポリシーページをレンダリングするために使われます。プライバシーポリシーページのテンプレートは、静的ページ（`page.php`）のテンプレートよりも優先されます。`privacy-policy.php` ファイルが存在しない場合、WordPress は利用可能なテンプレートに応じて `page.php` または `singular.php` ファイルを使用します。これらのファイルが存在しない場合は、`index.php`ファイルを使用します。